### PR TITLE
Fix/uppsf 864 temp hide hasbrand

### PR DESCRIPTION
**Pull request by epavlova**

### CODE REVIEW

File: annotations/models.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ var predicates = map[string]string{
 	"HAS_CONTRIBUTOR":            "http://www.ft.com/ontology/hasContributor",
 	"HAS_DISPLAY_TAG":            "http://www.ft.com/ontology/hasDisplayTag",
 	"IS_CLASSIFIED_BY":           "http://www.ft.com/ontology/classification/isClassifiedBy",
+	"HAS_BRAND":                  "http://www.ft.com/ontology/classification/isClassifiedBy",
```
Comment (karsov) [inline annotations/models.go]: I believe this should be "http://www.ft.com/ontology/hasBrand".

Comment (karsov) [reply annotations/models.go]: My mistake - we want to mock HAS_BRAND as "isClassifiedBy" for the time being.

File: annotations/fixtures/Content-ae17012e-ad40-11e9-8030-530adfa879c2.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "alternativeTitles": {
+    "promotionalTitle": "UK equities: the Boris Johnson trade"
+  },
+  "standfirst": "What if new Tory leader pulls Britain out of the EU relatively painlessly?",
+  "alternativeStandfirsts": {},
+  "byline": "",
+  "firstPublishedDate": "2019-07-23T14:58:19.000Z",
+  "publishedDate": "2019-07-23T14:58:19.000Z",
+  "brands": [
+    "http://api.ft.com/things/dbb0bdae-1f0c-11e4-b0cb-b2227cce2b54"
```
Comment (MiroslavGatsanoga) [inline annotations/fixtures/Content-ae17012e-ad40-11e9-8030-530adfa879c2.json]: This is none of the three concepts mentioned in the body field. Is this intentional?

Comment (epavlova) [reply annotations/fixtures/Content-ae17012e-ad40-11e9-8030-530adfa879c2.json]: No, it's not intentional. This is the id of the brand `Financial Times` on production. I'm replacing it with the id of `Financial Times` in the tests so it's not confusing. This field is actually not used in the tests btw.

**Comment by epavlova** [comment]
The good news is that we don't need to add `--build-tags=integration` when calling the linters. The code in the integrations tests is checked even without this flag. I think that only `--new-from-rev` is restricting which files will be checked.

**Pull request merged**